hrm-service/service-tests/case-validation.ts

```diff
@@ -123,7 +123,11 @@ export const validateSingleCaseResponse = (actual, expectedCaseModel, expectedCo
 };
 
 export const fillNameAndPhone = (
-  contact: NewContactRecord & { form?: ContactRawJson },
+  contact: Omit<NewContactRecord, 'rawJson'> & {
+    form?: any;
+    rawJson?: ContactRawJson;
```

Comment:
The original type required a 'rawJson' property. I wanted to replace it with one that could have 'form' or 'rawJson' as it's JSON blob property.

I could've defined a type which required one or the other to be present, but it made the code using it a lot more verbose to narrow the types, and it doesn't serve much purpose, since it would do nothing to stop a payload without either set being sent at runtime anyway